app/api/tutors/new/route.js

```diff
@@ -0,0 +1,21 @@
+import { db } from "@/firebase";
+
+export async function POST(request) {
+  const { userID, gender, bookingLink, ageGroup, subjects } =
+    await request.json();
+  let status = "success";
+  let tutor = {
+    gender: gender,
+    bookingLink: bookingLink,
+    ageGroup: ageGroup,
+    subjects: subjects,
+  };
+  try {
+    db.collection("tutors").doc(userID).set({ tutor });
+    return Response.json({ status, data: tutor });
```

Comment:
you can just directly put the success message here since the variable is not used elsewhere

```suggestion
    return Response.json({ status: "success" , data: tutor });
```